### DISCUSSION TITLE
dev-1.1.2进2.7白屏怎么解决？

### DISCUSSION THREAD

**Discussion by yoimiya43** [Melledy/Grasscutter#1163] [General]
点进入游戏就白屏，没有元素图标读条，cmd端就显示链接断开
![image](https://user-images.githubusercontent.com/106745510/171660016-612b9bec-22fd-41f9-833b-d07f23e570ac.png)

**Comment by XiFeng-exe** [comment]
启动器，设置，清空下载资源

**Reply by XiFeng-exe** [reply]
这个是针对logo后直接白屏的，开门白屏就比较麻烦了，你可以尝试更新一下jar文件，同时删除config.json

**Reply by yoimiya43** [reply]
好的谢谢我试一下
